### ISSUE TITLE
Make Upgrade button optional in Learner Dashboard

### ISSUE THREAD

**Issue by arbrandes** [openedx/frontend-app-learner-dashboard#297]

**Comment by arbrandes** [comment]
@jsnwesson, do I understand correctly that this is already queued up for "pluginification"?

**Comment by jsnwesson** [comment]
@arbrandes that's correct! This sprint (3/18 - 3/29) I'll be working on making sure we know how the Upgrade button logic works and what it'll need to be turned into a plugin.

**Comment by deborahgu** [comment]
we're preparing to schedule this work.

**Comment by jsnwesson** [comment]
Upgrade button is indeed optional as of #464 , and there is now current work to completely remove the Upgrade button from Learner Dashboard!
